utils/src/main/java/org/nervos/ckb/address/AddressUtils.java

```diff
@@ -23,16 +26,23 @@ public AddressUtils(Network network) {
     this.network = network;
   }
 
-  public String generate(String publicKey) {
+  public String generate(String publicKey) throws AddressFormatException {
     // Payload: type(01) | bin-idx("P2PH") | pubkey blake160
     String payload = TYPE + strToAscii(BIN_IDX) + blake160(publicKey);
     byte[] data = Numeric.hexStringToByteArray(payload);
-    return Bech32.encode(prefix(), convertBits(data, 8, 5, true));
+    return Bech32.encode(prefix(), convertBits(Bytes.asList(data), 8, 5, true));
+  }
+
+  public String generateFromArg(String arg) throws AddressFormatException {
```

Comment:
Two questions/issues:

1. What does `arg` mean?
2. `generate` and `generateFromArg` have most of the implementation duplicated. One should call another.